heartbeat/_meta/fields.common.yml

```diff
@@ -174,6 +174,45 @@
               type: text
             - name: stack
               type: text
+        - name: browser
+          type: group
+          fields:
+            - name: experience
+              type: group
+              fields:
+                - name: name
+                  type: keyword
+                - name: type
+                  type: text
+                - name: start
+                  type: long
+                  description: >
+                    duration relative to journey start in milliseconds
+            - name: user_timing
+              type: group
+              fields:
+                - name: name
+                  type: keyword
+                - name: type
```

Comment:
Can we document the `type`? It doesn't need to be exhaustive. I see `mark` in our test fixtures in synthetics, what are the other types?